app/assets/javascripts/pageflow/editor/utils/update_stylesheet.js

```diff
@@ -0,0 +1,11 @@
+pageflow.updateStylesheet = function(name) {
+  var link = $('head link[data-name=' + name + ']');
+
+  var newHref = pageflow.editor.themes.find(function(theme) {
+    return theme.attributes.name == pageflow.entry.configuration.attributes.theme_name;
+  }).get('stylesheet_path');
+
+  if (link.attr('href') !== newHref) {
+    link.attr('href', newHref);
+  }
+};
```

Comment:
Same goes for the logic that finds a theme by name, which is also repeated in a couple of places.